plugins/node/opentelemetry-instrumentation-express/README.md

```diff
@@ -79,6 +79,13 @@ Express instrumentation has few options available to choose from. You can set th
 - `info: ExpressRequestInfo` containing the incoming Express.js request, the current route handler creating a span and `ExpressLayerType` - the type of the handling layer.
 - `defaultName: string` - original name proposed by the instrumentation.
 
+`requestHook` is invoked with 2 arguments:
+
+- `span: Span` - the span associated with the express request.
+- `info: ExpressRequestInfo` containing the incoming Express.js request, the current route handler creating a span and `ExpressLayerType` - the type of the handling layer.
+
+NOTE: `ExpressRequestInfo.request` is typed as `any`. If you want type support make sure you have `@types/express` installed then you can use `ExpressRequestInfo<express.Request>`
```

Comment:
Just to make sure I got it right - so a user looking for hook typing, can use:

```js
import type { Request } from 'express';
import { ExpressRequestInfo } from '@opentelemetry/instrumentation-express';

const spanNameHook = (info: ExpressRequestInfo<express.Request>, defaultName: string): string => {
//....
}
```

Right?

Reply:
that's correct @blumamir :)

by doing so devs will have the full type with the current version of `@types/express` they have installed in their app. Otherwise the property is typed as `any` which will not harm TS compilation.